.github/workflows/build-commit.yml

```diff
@@ -26,4 +30,6 @@ jobs:
         with:
           dotnet-version: 8
       - name: Build Project
-        run: dotnet build
\ No newline at end of file
+        run: dotnet build
```

Comment:
(potentially for future reference)
ideally we want to pass the run number and hash here so that we can build assembly metadata from them if/when we add dev builds. not a priority right now though